joint_teapot/utils/joj3.py

```diff
@@ -201,13 +201,17 @@ def generate_title_and_comment(
         comment += "\n"
         for i, result in enumerate(stage["results"]):
             if not single_case:
-                comment += f"### Case {i} - Score: {result['score']}\n"
+                comment += (
+                    f"<details><summary>Case {i} - Score: {result['score']}</summary>\n"
+                )
+            else:
+                comment += "<details><summary>Details</summary>\n"
```

Comment:
What about showing the score as the summary and removing line 199 and 200?

Reply:
For consistency, maybe I'll just remove the single_case variable?